docs/docs/deployment/pre_deployment/prerequisites.md

```diff
@@ -16,5 +16,4 @@ During the installation, we will require access to the billing account (Created
 
 ## Cost Explorer
 
-In order to see any actual cost in dashboards and workspaces, the master account must have Cost Explorer set up. 
```

Comment:
This line would be a prerequisite to the one you've added, so it should still exist.